project_task_schedule/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 Sunflower IT <https://sunflowerweb.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Project Task Schedule",
+    "version": "13.0.1.0.1",
```

Comment:
13.0.1.0.0

----
Internal review first, but this looks nice, I think we can directly propose it to OCA and ping Pedro; in `https://github.com/OCA/project/issues/768` he is proposing to add the fields to `project_timeline` instead of a new module.